src/client/ContentAccess.js

```diff
@@ -1147,6 +1147,60 @@ exports.LatestVersionHashV2 = async function({objectId, versionHash}) {
   return latestHash;
 };
 
+
+/**
+ * Check to see if version hash is available.
+ * Used after finalizing a draft to determine if publishing has finished.
+ *
+ * @methodGroup Content Objects
+ * @namedParams
+ * @param {string=} versionHash - Version hash of the object
+ *
+ * @returns {Promise<boolean>} - Whether or not the version hash is visible
+ */
+exports.VersionAvailable = async function({versionHash}) {
+  this.Log(`Checking if version is available: ${versionHash}`);
+
+  ValidateParameters({versionHash});
+
+  const status = await VersionStatus({versionHash});
+  return status.available;
+};
+
+/**
+ * Get status of version hash
+ *
+ * @methodGroup Content Objects
+ * @namedParams
+ * @param {string=} versionHash - Version hash of the object
+ *
+ * @returns {Promise<object>} - Information about the version hash
+ */
+exports.VersionStatus = async function({versionHash}) {
+  this.Log(`Querying status of version: ${versionHash}`);
+
+  ValidateParameters({versionHash});
+
+  // For now, no auth required for this call
+
+  // const authToken = await this.authClient.AuthorizationToken({
+  //   noAuth: true  // Don't create an accessRequest blockchain transaction
+  // });
+
+  return await this.utils.ResponseToJson(
+    this.HttpClient.Request({
+      path: UrlJoin("q", versionHash, "status"),
+      method: "GET"
+      // headers: {
+      //   Authorization: `Bearer ${authToken}`
```

Comment:
Here we need to use the same token we used for editing the object.  The auth token has already been created in this client so it should be used form cache.  I believe authClient.AuthorizationToken() doesn find it in cache but I want to make sure that if we have an update token in cache we use it here weather or not we request an update token.
In general we should use the update token if we have one available in cache instead of making an access request.

Reply:
This could potentially be called by a new ElvClient instance - if a script were just checking for existence of a VersionHash (or access to it). I've changed to use same auth token logic as `ContentObject()`, which should pull from client cache if available.